.github/FAQ.md

```diff
@@ -2,17 +2,19 @@
 
 <!-- toc -->
 
-- [What does _evergreen_ mean?](#what-does-_evergreen_-mean)
-- [The `[::]` URI is odd, what is that?](#the--uri-is-odd-what-is-that)
-- [Why doesn't the server default to `localhost`?](#why-doesnt-the-server-default-to-localhost)
-- [What does the "Not Found / 404" error mean?](#what-does-the-not-found--404-error-mean)
-- [Why does the `static` option default to `compiler.context`?](#why-does-the-static-option-default-to-compilercontext)
-- [Why can't I use Node v10.14.x?](#why-cant-i-use-node-v1014x)
-- [Why do I have to include `webpack-plugin-serve/client` in the entry?](#why-do-i-have-to-include-webpack-plugin-serveclient-in-the-entry)
+- [Frequently Asked Questions](#frequently-asked-questions)
```

Comment:
Can you revert back from what it was before? It always happens when you save MD files on vscode specially.

Reply:
Oops I'll fix that. Sorry!